packages/shared/prisma/schema.prisma

```diff
@@ -406,7 +406,7 @@ model Score {
     observationId String?        @map("observation_id")
     configId      String?        @map("config_id")
     stringValue   String?        @map("string_value")
-    dataType      ScoreDataType  @default(NUMERIC) @map("data_type")
+    dataType      ScoreDataType? @default(NUMERIC) @map("data_type")
```

Comment:
non-null enum with default works without backfill. you'd need to regenerate the migration here. will work out of the box

default value will be written to metadata of column in postgres this way, thus needs no backfill. on read, behaves in the same way as if we did a backfill

Reply:
Super cool, did not know. Thanks!